main.c

```diff
@@ -6,7 +6,7 @@
 
 
 int luaopen_buffer(lua_State *L);
-int luaopen_mpi(lua_State *L);
+int luaopen_MPI(lua_State *L);
```

Comment:
The na,e in upper case allow to do require 'MPI' from plain Lua binary executable.